src/Microsoft.VisualStudio.Threading.Tests/ReentrantSemaphoreTestBase.cs

```diff
@@ -227,7 +228,7 @@ public void ExitInAnyOrder_ExitInAcquisitionOrder()
         this.semaphore = this.CreateSemaphore(ReentrantSemaphore.ReentrancyMode.Freeform);
         this.ExecuteOnDispatcher(async delegate
         {
-            var releaser1 = new AsyncManualResetEvent();
+            var outerReleaser = new AsyncManualResetEvent();
```

Comment:
>outerReleaser [](start = 16, length = 13)

So... this change seems to be to a variable that no one uses. Shall we revert the change, or simply delete the line?